src/TetherComponent.jsx

```diff
@@ -30,7 +30,7 @@ const attachmentPositions = [
 class TetherComponent extends Component {
   static propTypes = {
     renderElementTag: PropTypes.string,
-    renderElementTo: PropTypes.string,
+    renderElementTo: PropTypes.any,
```

Comment:
I think it should either be a string or a DOM node. So i'm guessing we would need a custom proptype checking if it's a DOM element? Something like this http://stackoverflow.com/a/385427/1461204